inst/workflow/1_mappings/STUDY.yaml

```
@@ -43,54 +43,54 @@ spec:
       type: character
 steps:
   - output: Temp_CTMSStudyWide
-    name: RunQuery
+    name: gsm::RunQuery
     params:
       df: Raw_STUDY
       strQuery: "SELECT studyid as GroupID, * FROM df"
   - output: Temp_CTMSStudy
-    name: MakeLongMeta
+    name: gsm.mapping::MakeLongMeta
     params:
       data: Temp_CTMSStudyWide
       strGroupLevel: "Study"
   - output: Temp_CTMSplanned
-    name: RunQuery
+    name: gsm::RunQuery
     params:
       df: Raw_STUDY
       strQuery: "SELECT studyid as GroupID, num_plan_site as SiteTarget, num_plan_subj as ParticipantTarget FROM df"
   - output: Temp_StudyCountsWide
-    name: RunQuery
+    name: gsm::RunQuery
     params:
       df: Mapped_SUBJ
```

```suggestion
      df: Raw_SUBJ
```

same reference as Line 37